File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntReportQuery.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdSpreadsheet;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioEntReportQuery extends StudioEntReport
{
  @Nullable
  public MetaIdSpreadsheet[] fromSpreadsheetIdSet;

  @Nullable
  public StudioValueCodeNeoQL neoQL;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntReportQuery.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdSpreadsheet
import com.neome.api.meta.base.dto.StudioEntReport
import com.neome.api.meta.base.dto.StudioValueCodeNeoQL

interface StudioEntReportQuery : StudioEntReport
{
  val fromSpreadsheetIdSet: List<MetaIdSpreadsheet>?
  val neoQL: StudioValueCodeNeoQL?
}